src/app/containers/CpsAssetPageMain/Blocks/Video/__snapshots__/index.test.jsx.snap

```diff
@@ -0,0 +1,5 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`CPSVideoBlock no placeholder 1`] = `<div />`;
```

Comment:
Whats this?

Reply:
Me forgetting to update unit tests 🤕  fixed